lib/l10n/app_nl.arb

```diff
@@ -128,5 +128,6 @@
   "experimentalDescription": "Deze functie is experimenteel. Er kunnen dingen misgaan.",
   "usernamePatternError": "Waarde mag alleen letters, cijfers, streepjes en underscores bevatten.",
   "genericError": "Er ging iets mis",
-  "wittyError": "Drommels, drommels, en nog eens drommels"
+  "wittyError": "Drommels, drommels, en nog eens drommels",
+  "toggleVisited": "Toggle Bezocht"
```

Comment:
Following the suggestions for the English strings, the Dutch equivalents would be "Markeer als gelezen" and "Markeer als ongelezen" respectively.